ptc/reset_camera_button.py

```diff
@@ -99,7 +97,7 @@ def change_interaction_mode(mode: Literal["2D", "3D"]) -> None:
                         icon="mdi-crop-free",
                         v_bind=("props",),
                         click=reset_camera_callback,
-                        v_if=("reset_camera_visibility",),
+                        v_if=("show_reset_camera",),
```

Comment:
instead of having `setdefault` above, just inline the default value in the `v_if=("show_reset_camera", show_reset_camera)`.

Sorry for not mentioning it before, but I though it was because you had the UI definition in a different scope so you could not use the variables from the constructor. 

Also should that be a v_if or a v_show? Should that be on the tooltip subtree?